Services/Sessions/WebAppSession.m

```diff
@@ -299,4 +299,28 @@ - (ServiceSubscription *)subscribeMediaInfoWithSuccess:(SuccessBlock)success fai
     return nil;
 }
 
+#pragma mark - Playlist Control
+
+- (id <PlayListControl>)playListControl
+{
+    return nil;
+}
+
+- (CapabilityPriorityLevel)playListControlPriority
+{
+    return CapabilityPriorityLevelVeryLow;
```

Comment:
According to https://github.com/ConnectSDK/Connect-SDK-iOS-Core/blob/master/Services/Capabilities/Capability.h#L24:
> Values of VeryLow and VeryHigh are not in use internally the SDK.

I think we should change it to Low.